tests/devices/test_lightning_qubit.py

```diff
@@ -89,42 +89,40 @@ def circuit():
     assert np.allclose(circ0(), circ1(), rtol=0.01)
 
 
-@pytest.mark.skip(reason="New lightning.qubit needs to be updated to work with dtypes")
```

Comment:
Are the changes in this file related to the PR immediately, or more of a byproduct?

Reply:
It's a byproduct of changing `LightningQubit` to use the new API. Somewhere along processing measurements, the dtype gets converted to double precision. This needs to be investigated, but that's a task for another PR.